### ISSUE TITLE
Second UK Delta/BA.1 recombinant lineage under monitoring in the UK

### ISSUE THREAD

**Issue by thomasppeacock** [cov-lineages/pango-designation#441]
**New proposed recombinant lineage**

**Description
Recombinant cluster between:** AY.4 x BA.1
**Earliest Sequence:** 2022-01-17
**Latest Sequence:** 2022-01-22

**Countries circulating:** UK (5 genomes)

**Description:**

**Likely Break point:** Near start of Spike protein

**Conserved Spike mutations -** T19R /// A67V, Δ69-70, T95I, G142D/Δ143-145, Δ211/L212I, ins214EPE, G339D, S371L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, A701V, N764K, D796Y, N856K, Q954H, N969K, L981F (/// = proposed break point)

**Conserved non-Spike mutations -** NSP3 –A488S, P1288L, R1341W, P1469S, A1711V; NSP4 – V167L, T492I; NSP6 – T77A; NSP12 - P323L, G671S; NSP13 – P77L; NSP14 – A394V; NSP16 – L212F; E – T9I; M – D3G, Q19E, A63T; N – P13L, Δ31-33, R203K, G204R

Genomes:
EPI_ISL_9125312
EPI_ISL_9175215
EPI_ISL_9026013
EPI_ISL_9062592
EPI_ISL_9296739

Evidence:
![image](https://user-images.githubusercontent.com/79373149/154043520-cd3c6236-ff86-4cb0-9a3e-b78770237a67.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_41fc0_b80290.json?c=pango_lineage_usher&label=nuc%20mutations:C6402T,C7124T

Proposed lineage name: Next available X recombinant lineage

**Comment by thomasppeacock** [comment]
Also just confirming here that the other recombinant cluster under UKHSA monitoring is the same one described in issue #422

**Comment by thomasppeacock** [comment]
@drmutaba yes we had also noticed that! @c19850727 has had a look at all the singlet recombinants as well and says the pattern mostly holds up there as well  (ie 5' Delta, 3' BA.1, breakpoint somewhere between the start of NSP14 and the end of the Spike NTD)

**Comment by AngieHinrichs** [comment]
> breaking points of all sequences found so far seem to be at/near beginning of the spike

That's consistent with what my colleagues found in a search for recombinants in an earlier version of the UCSC/UShER tree (back before amplicon dropout & contamination problems drove us to filter out partial-Omicron sequences including potential recombinants, regretfully):

[Pandemic-Scale Phylogenomics Reveals Elevated Recombination Rates in the SARS-CoV-2 Spike Region (bioRxiv)](https://www.biorxiv.org/content/10.1101/2021.08.04.455157v1.full)
